test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -5507,30 +5475,38 @@ def forward(self, input, target):
         target = torch.empty(N, 8, 8, dtype=torch.long).random_(0, C)
         self.run_test(NLLModel(), (input, target))
 
-    @disableScriptTest()
     @skipIfUnsupportedMinOpsetVersion(12)
     def test_binary_cross_entropy_with_logits(self):
         x = torch.randn(5)
         y = torch.empty(5).random_(2)
         self._bce_logits_loss(x, y)
 
-        x = torch.randn(2, 3, 5, 7)
```

Comment:
We need 4 variants of this test:
bce_logits
bce_logits_wegiht
bce_logits_posweight
bce_logits_weight_posweight

Please do not delete any tests. This is critical.